internal/langserver/handlers/handlers_test.go

```diff
@@ -71,6 +71,11 @@ func initializeResponse(t *testing.T, commandPrefix string) string {
 						"supported": true,
 						"changeNotifications": "workspace/didChangeWorkspaceFolders"
 					}
+				},
+				"experimental": {
+					"referenceCountCodeLens": false,
+					"refreshModuleProviders": false,
+					"refreshModuleCalls": false
```

Comment:
Was there a particular motivation behind adding these? In Go fields of primitive types generally default to the "zero" - such as `false`.

Reply:
Oh right! Now I understand this relates to the bug you fixed in `initialize.go` as we now _always_ send all the experimental capabilities with their default values. That makes more sense now! Thanks for that context.